scripts/build-in-docker.sh

```diff
@@ -15,7 +15,10 @@ DOCKER_RUN_OPTS="
 
 docker run \
   ${DOCKER_RUN_OPTS} \
-  cargo build --release --target=${TARGET} --locked
+  /bin/bash -c "rustup toolchain install nightly;
+  rustup component add rust-src --toolchain nightly;
+  cargo +nightly run -Z build-std --target ${TARGET}; 
+  RUSTFLAGS=\"-C force-frame-pointers=yes\" cargo build --release --target=${TARGET} --locked"
```

Comment:
In the CI job you use `override` which means `cargo +nightly` is the same as `nightly`, but I think here the second command is actually using stable Rust vs nightly.
Not quite sure how exactly `build-std` is supposed to work, I would need to look that up.

Reply:
When I ran my experiments (both on Intel Mac and Linux VM) that seemed to work fine.

I'd run the second command without first enabling the nightly toolchain but simply using the stable one.